db/seeds.rb

```diff
@@ -35,3 +35,7 @@
 FileStorage.default = FileStorage::LocalFile.new(path: Rails.root.join('db'))
 ImportVersionsJob.perform_now(import)
 FileStorage.default = fs_default
+
+puts "\n\n------------------------------------------------------------"
+puts "Login in with these admin credentials: #{admin.email} / #{admin.password}"
+puts "------------------------------------------------------------\n\n"
```

Comment:
Hmmm. A few concerns here:

1. `admin.password` will always return `nil` (it’s just a setter that eventually modifies `encrypted_password`), so this line won’t give you useful credentials.
2. At this point, there’s actually no way to get the password, so you actually *can’t* print usable credentials here (we only store the hashed password).
3. If a user was created, we have sort-of-duplicated output (from line 24), which is weird/confusing.
4. Do we really need to print anything here if they already had a user present and we didn’t actually create one?

Reply:
Ah sorry, I could've sworn I saw this work. Definitely not committed to this way I've done it.

As for why it's helpful here: We have another PR that prints feedback on the sometimes long/silent process of importing: https://github.com/edgi-govdata-archiving/web-monitoring-db/pull/293

I felt that any credentials should be printed at the end of that whole process, when control is kicked back to user. But yes, only one place is necessary.